packages/core/src/server/index.ts

```diff
@@ -128,6 +128,7 @@ export class Server extends httpServer {
    */
   constructor(readonly inlineConfig: FarmCliOptions & UserConfig) {
     super();
+    this.logger = new Logger();
```

Comment:
Honestly, I'm not quite sure what this line of code does, it looks like we're correctly creating a logger when we call `createServer` anyway, is it really necessary here?